Integrations/integration-Zendesk.yml

```diff
@@ -511,17 +522,23 @@ script:
             ec.Ticket = newContext;
         }
 
-
         var res = {
             TicketDetails: resDetails,
             TicketComments: resComments,
             Users: usersEc,
+            Org: org,
             CustomFields: fieldsEc
         }
-
         return({ContentsFormat: formats.json, Type: entryTypes.note, Contents: res, HumanReadable: md, EntryContext: ec})
     };
 
+    var getArticle = function() {
+        var locale = (args.locale) ?  args.locale : 'en-us';
+        var id = args.articleID;
+        var url = 'help_center/' + locale + '/articles/' + id + '.json';
+        res = sendRequest('GET', url);
+        return ( {ContentsFormat: formats.html, Type: entryTypes.note, Contents: res.article.body} );
```

Comment:
just pay attention that here you expose to XSS attacks.. but because our server handles XSS we should be fine. - please verify that.

Reply:
good point. will check

Reply:
Checked.
Looks like zendesk themselves do not allow XSS:
> Zendesk Support utilizes Ruby on Rails framework security controls to limit exposure to OWASP Top 10 security flaws. These include inherent controls that reduce our exposure to Cross Site Scripting (XSS), Cross Site Request Forgery (CSRF), and SQL Injection (SQLi), among others.

In addition, I checked manually and we also handle it well.